ui/src/app/views/concept-set-list/component.tsx

```diff
@@ -0,0 +1,123 @@
+import {Component} from '@angular/core';
+import * as React from 'react';
+
+import {
+  DomainInfo,
+  RecentResource,
+  WorkspaceAccessLevel,
+} from 'generated/fetch';
+
+import {conceptsApi, conceptSetsApi} from 'app/services/swagger-fetch-clients';
+import {reactStyles, ReactWrapperBase, withCurrentWorkspace} from 'app/utils/index';
+
+import {CreateConceptSetModal} from 'app/views/conceptset-create-modal/component';
+
+import {CardButton} from 'app/components/buttons';
+import {ClrIcon} from 'app/components/icons';
+import {TooltipTrigger} from 'app/components/popups';
+import {SpinnerOverlay} from 'app/components/spinners';
+import {WorkspaceData} from 'app/services/workspace-storage.service';
+import {convertToResources, ResourceType} from 'app/utils/resourceActions';
+import {ResourceCard} from 'app/views/resource-card/component';
+
+const styles = reactStyles({
+  pageArea: {
+    display: 'flex',
+    justifyContent: 'flex-start'
+  },
+  resourceCardArea: {
+    display: 'flex',
+    flexWrap: 'wrap'
+  }
+});
+
+export const ConceptSetsList = withCurrentWorkspace()(
+  class extends React.Component<{workspace: WorkspaceData},
+      {conceptSetsList: RecentResource[], conceptSetsLoading: boolean,
+        conceptDomainsList: Array<DomainInfo>, createModalOpen: boolean}> {
+    constructor(props) {
+      super(props);
+      this.state = {
+        conceptSetsList: undefined,
+        conceptSetsLoading: true,
+        conceptDomainsList: undefined,
+        createModalOpen: false
+      };
+    }
+
+    componentDidMount() {
+      try {
+        this.getDomainInfo();
+      } finally {
+        this.loadConceptSets();
+      }
+    }
+
+    async getDomainInfo() {
+      const {namespace, id} = this.props.workspace;
+      try {
+        const resp = await conceptsApi().getDomainInfo(namespace, id);
+        this.setState({conceptDomainsList: resp.items});
+      } catch (error) {
+        console.log(error);
```

Comment:
what do you mean by this?  do you have an example you can share?  (this is how we've been handling most api calls that could potentially error, though it may need to be revisited. seems like we are just logging and then swallowing the error?)

Reply:
yeah the existing error handler component doesn't usually give an error message (unless it's a popup error).  if it's the red banner thing it will just say "500 error" and leave it at that.